bylaws/009-funding.md

```diff
@@ -0,0 +1,47 @@
+# Funding
+
+## Scope and objectives
+
+PHP-FIG is, as it often happens in Open Source communities, an organization composed by volunteers only; this doesn't exclude the fact that the organization itself may have some costs to sustain itself, like for maintaining a couple of tools such as a site and an email account. For such and similar reasons, we need a way to raise and govern money inside our organization.
+
+This document defines the roles, rules and processes that must be followed inside the PHP-FIG in all matters regarding money, to guarantee fairness, transparency and impartiality. This document intentionally describes the only allowed processes and ways to raise, administer and spend money, so that any change to such processes or any variation in budget allocation requires a bylaw change vote. 
+
+## Fiscal hosting
+For handling money and fiscal matters, the PHP-FIG will use [OpenCollective](https://opencollective.com/) as a fiscal host.
+As for any other tool pertaining to the PHP-FIG, secretary will be the only ones to have full administrative access to PHP-FIG's OpenCollective account, but they will not have any decision power over it, and they are instead bounded by this document.
+
+All monetary contributions will flow through it, so that any tax matter will be handled automatically, and every financial decision will be published and publicly available through their budgetary tools.
+
+## Raising money
+### Through OpenCollective
+Contributions will be accepted through OpenCollective; PHP-FIG members will refrain from suggesting or requiring contributions from other members or the community at large if the monetary needs of the PHP-FIG are already met (see "Spending money" section).
+
+### Through TideLift
+As the main way of raising money, PHP-FIG will use [TideLift](https://tidelift.com/).
+The TideLift account will pay out any dividend to the OpenCollective account.
+
+## Spending money
+Any payment has to be budgeted and described in this document. Any payment has to be justified and aligned with the mission of the PHP-FIG, which is to push forward the evolution of the PHP community, through the definition of common standard and tools, and by helping PHP Open Source maintainers which follow the same goals.
+
+### Overfunding, and giving back to the community
+PHP-FIG is allowed to store in its OpenCollective account enough funds to sustain itself for three years in the future, according to the current approved budget.
+At least once a year, any exceeding funds will be forwarded to the [PHP Foundation](https://opencollective.com/phpfoundation), as a way to give back to the community, and further increase its growth.
+
+### Approved expenses
+This section will enumerate any approved expense.
```

Comment:
I am skeptical of making the budget part of the bylaw.  That means any new budgetary change requires a full vote of all project reps as well as the CC, and... well, projects are not exactly active.  I'd prefer to have the budget by specifically as specified by the CC, not a full bylaw change.

We can wordsmith that if you're OK with that approach.

Reply:
Done in de5bbe8